scripts/modifyConfigIfRCVersion.js

```diff
@@ -12,11 +12,13 @@ if (process.env.TRAVIS_TAG) {
     packageJson.name = testName;
     packageJson.displayName = testDisplayName;
     packageJson.publisher = testPublisher;
+    packageJson.aiKey = process.env['TEST_AIKEY'];
```

Comment:
do we want to check if AI Key is not a placeholder here?

Reply:
No. We will replace the "aikey" field in `package.json` with `TEST_AIKEY` if RC tag. And replace with `PROD_AIKEY` if production tag.